File: app/src/main/java/com/skripsi/presensigps/network/model/Report.kt
package com.skripsi.presensigps.network.model

class ReportIndexModel(
    val success: Boolean,
    val message: String,
    val data: ArrayList<DataReportModel>,
)

class ReportAddModel(
    val success: Boolean,
    val message: String,
    val data: DataReportModel,
)

class ReportDeleteModel(
    val success: Boolean,
    val message: String,
)

class ReportVerifyModel(
    val success: Boolean,
    val message: String,
)

data class DataReportModel(
    val id: Int,
    val user_id: Int,
    val name: String,
    val latitude: String,
    val longitude: String,
    val note: String,
    val image: String,
    val status: Int,
    val created_at: String,
)

File: backend/presensi-gps/app/Models/Report.php
<?php

namespace App\Models;

use Illuminate\Contracts\Auth\MustVerifyEmail;
use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Factories\HasFactory;

class Report extends Model
{
    use HasFactory;

    protected $table = 'reports';

    protected $fillable = [
        'user_id','name','latitude','longitude','note','image','status','created_at'
    ];

}